Empty.bot

```diff
@@ -0,0 +1,17 @@
+// Fills an empty room
+// Also demonstrates and tests most language features
+ _      1  |  #      #  |  #      # // This is a comment
```

Comment:
Oh man, you guys got comments working! I wish we'd done that too, it seems to me like the ability to have comments is an essential feature of a language.